README.md

```diff
@@ -232,6 +232,7 @@ Once you have a development environment set up, you might find the following use
 | Build test coverage report | `pytest --cov-report html:cov_html --cov=./` | `pipenv run pytest --cov-report html:cov_html --cov=./` |
 | Check type annotations | `./tools/run-mypy` | `pipenv run ./tools/run-mypy` |
 
+If you plan to add git commits and submit pull-requests (PRs), then we highly suggest installing the `gitlint` commit-message hook by running `gitlint install-hook` (or `pipenv run gitlint install-hook` with pipenv setups). While the content still depends upon your writing skills, this ensures a more consistent formatting between different commits by different authors.
```

Comment:
We should also add instructions to run it properly here.

Reply:
Do you mean describing that it will check and maybe ask the yes/no/edit question?
Could you describe more what you mean by this?

Reply:
I mean that we should specify gitlint will check commit message after every `git commit XYZ` command. Also add some commands to check commit message lints for last n commits.

Reply:
I've pushed a new version that should address these points. Let me know if they cover what you expect.